include/sddf/i2c/queue.h

```diff
@@ -45,8 +46,9 @@ enum i2c_token {
     I2C_TOKEN_ADDR_WRITE = 0x2,
     /* ADDRESS READ: Same as ADDRW but sets up DATA tokens as reads. */
     I2C_TOKEN_ADDR_READ = 0x3,
-    /* DATA_LAST: Used for read transactions to write a NACK to alert
-     * the slave device that the read is now over. */
+    /* DATA_END: Used for read transactions to write a NACK to alert
```

Comment:
This is not correct for semantics. This token represents the end of a read operation but not the end of an operation in general

Reply:
I just added this bit "this also acts the same way as I2C_TOKEN_DATA (ie it actually requests a token to read so should be treated the same in that regard)" the other part about NACKing was already there